lib/spack/spack/solver/display.lp

```diff
@@ -9,26 +9,10 @@
 % This section determines what parts of the model are printed at the end
 %==============================================================================
 
-% Spec-related functions.
-% Used to build the result of the solve.
-#show node/1.
-#show hash/2.
-#show depends_on/3.
-#show version/2.
-#show variant_value/3.
-#show node_platform/2.
-#show node_os/2.
-#show node_target/2.
-#show node_compiler/2.
-#show node_compiler_version/3.
-#show node_flag/3.
-#show node_flag_compiler_default/1.
-#show node_flag_source/3.
-#show no_flags/2.
-#show external_spec_selected/2.
-#show version_equivalent/3.
-
-#show build/1.
+% Spec attributes
+#show attr/2.
+#show attr/3.
+#show attr/4.
```

Comment:
Is there any way to print only `attr/N` atoms which match a certain first argument? Before in complex debugging situations we could comment all but a few atoms. I wonder if now, that are all aggregated under the same name, we can do something similar.

Reply:
Unfortunately, there isn't.